tests/forcetorque/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+project(GTestSetup)
+
+# Find Gazebo
+find_package(gz-sim7 REQUIRED)
+set(GZ_SIM_VER ${gz-sim7_VERSION_MAJOR})
```

Comment:
These lines are not requested, and can be removed to reduce the noise in the code. `project` is only required in the root `CMakeLists.txt` of the project, and `find_package` needs only to be invoked once. 
```suggestion
```